app/models/manageiq/providers/nuage/network_manager/event_catcher/messaging_handler.rb

```diff
@@ -0,0 +1,41 @@
+require 'qpid_proton'
```

Comment:
This cannot be in the top-level context.  As the gem is not required to be installed, this class will blow up on load (even if the worker is not started).  You can avoid this by just putting the require in the initialize call.